### PR TITLE
Correccionletras

### PR DESCRIPTION
**Pull request by RatamaG**
letter n made in uppercase with diagonal

### CODE REVIEW

**Comment by RatamaG** [comment]
Letraw y letrax creadas

File: cmd/letraw/main.go [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+func main() {
+	if len(os.Args) < 2 {
+		fmt.Println("Ingresa un numero ")
+
+		return
+	}
+	name := os.Args[1]
+	dim, err := strconv.Atoi(name)
+	if err != nil {
+		fmt.Println("ingresa un numero, letras no son compatibles")
```
Comment (paganotoni) [inline cmd/letraw/main.go:18]: Here you should return.

Comment (paganotoni) [inline cmd/letraw/main.go:18]: This message should be in English.

File: cmd/letraw/main.go [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+	}
+	for i := 0; i <= dim-1; i++ {
+		fmt.Print("*")
+		for j :=1; j <= dim-i; j++ {
+			fmt.Print(" ")
+		}
+		fmt.Print("*")
```
Comment (paganotoni) [inline cmd/letraw/main.go:25]: You should add spacing here. TO separate this into blocks.

File: cmd/letraw/main.go [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+	"strconv"
+)
+
+func main() {
+	if len(os.Args) < 2 {
+		fmt.Println("Ingresa un numero ")
+
+		return
+	}
+	name := os.Args[1]
```
Comment (paganotoni) [inline cmd/letraw/main.go:15]: Spacing needs to be added here after the block.

**Pull request merged**